apps/services/routers.py

```diff
@@ -14,7 +14,6 @@
 from apps.services.handlers import service_handler
 from apps.services.schemas import (
     CancelRequestIn,
```

Comment:
Should we delete `CancelRequestIn` if we replaced it by `CreateUserRequestIn`. As we did with `CancelRequestOut`&

Reply:
I can't find where it uses

Reply:
Ok